tests/ha/control_pod/test_control_pod_soft_failures.py

```diff
@@ -603,3 +620,180 @@ def test_iam_cruds_ios_during_soft_failure_all_pods(self):
+                                                    log_prefix=self.test_prefix, skipcleanup=True,
+                                                    setup_s3bench=False, nsamples=2, nclients=2)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 3: Performed IOs on already created buckets.")
+
+        LOGGER.info("Step 4: Create multiple buckets and perform IOs for the already created IAM "
+                    "user.")
+        self.test_io_prefix = 'test-45501-io'
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(self.s3_clean.values())[0],
+                                                    log_prefix=self.test_io_prefix,
+                                                    skipcleanup=True, setup_s3bench=False,
+                                                    nsamples=2, nclients=2)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 4: Create multiple buckets and perform IOs for the already created IAM "
+                    "user.")
+
+        LOGGER.info("Step 5: Create IAM user and perform IOs after soft-failure.")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.test_prefix = 'test-45501-1'
+        self.s3_clean.update(users)
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix, skipcleanup=True,
+                                                    nclients=2, nsamples=2, setup_s3bench=False)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("Step 5: Created IAM user and performed IOs after soft-failure.")
+        LOGGER.info("ENDED: Verify IAM user creation & IOs before and after soft-failures.")
+
+    @pytest.mark.ha
+    @pytest.mark.lc
+    @pytest.mark.tags("TEST-45502")
+    def test_iam_cruds_during_soft_failure_random_n_pods(self):
+        """
+        Verify IAM user creation/deletion and get IAM user list while soft-failure in loop.
+        """
+        LOGGER.info("STARTED: Verify IAM user creation/deletion and get IAM user list while "
+                    "soft-failure in loop.")
+        LOGGER.info("Precondition: Create %s iam users for deletion", self.num_users)
+        del_users = self.mgnt_ops.create_account_users(nusers=self.num_users)
+
+        if self.scale_req:
+            LOGGER.info("Scale replicas for control pod to %s", self.repl_num)
+            resp = self.node_master_list[0].create_pod_replicas(num_replica=self.repl_num,
+                                                                deploy=self.deploy_name)
+            assert_utils.assert_true(resp[0], resp[1])
+            LOGGER.info("Total number of control pods in the cluster are: %s", self.repl_num)
+
+        LOGGER.info("Step 1: Create multiple IAM users.")
+        new_users = self.mgnt_ops.create_account_users(nusers=10)
```

Comment:
use config instead

Reply:
Default value is 10 in function itself. We don't need to pass anything now.